apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/index.tsx

```diff
@@ -0,0 +1,333 @@
+import { shell, WebviewTag } from "electron";
+import * as remote from "@electron/remote";
+import { JSONRPCRequest } from "json-rpc-2.0";
+import React, { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch, useSelector } from "react-redux";
+import styled from "styled-components";
+
+import { listSupportedCurrencies } from "@ledgerhq/live-common/currencies/index";
+import { useToasts } from "@ledgerhq/live-common/notifications/ToastProvider/index";
+import { AppManifest } from "@ledgerhq/live-common/platform/types";
+import { useJSONRPCServer } from "@ledgerhq/live-common/platform/JSONRPCServer";
+import {
+  RawPlatformSignedTransaction,
+  RawPlatformTransaction,
+} from "@ledgerhq/live-common/platform/rawTypes";
+
+import TrackPage from "../../analytics/TrackPage";
+import useTheme from "../../hooks/useTheme";
+import { accountsSelector } from "../../reducers/accounts";
+import BigSpinner from "../BigSpinner";
+import Box from "../Box";
+
+import TopBar from "./TopBar";
+import * as tracking from "./tracking";
+import { TopBarConfig } from "./type";
+import {
+  listAccountsCallback,
+  listCurrenciesCallback,
+  receiveOnAccountCallback,
+  requestAccountCallback,
+  signTransactionCallback,
+  broadcastTransactionCallback,
+  startExchangeCallback,
+  CompleteExchangeRequest,
+  completeExchangeCallback,
+  RequestAccountParams,
+  signMessageCallback,
+} from "./LiveAppSDKCallback";
```

Comment:
I'm not sure callback is best wording for the implementation of api functions (desktop or mobile)

I would go with `Logic` or `Implementation` (`Implem` or `Impl` ?), `Logic` seems the best as it's shorter than `Implementation` and more explicit than `Impl`

WDYT ?